org.openhab.binding.zigbee/README.md

```diff
@@ -38,6 +38,14 @@ The Ember EZSP NCP (Network Co-Processor) supports the Silabs EM358 or MightyGec
 
 The thing type is ```coordinator_telegesis```.
 
+#### Examples for coordinators
```

Comment:
Thanks again for this.

Can I suggest that instead of putting this list at the end of the coordinators, that it might be better to put the examples of each coordinator into the coordinator section? At the moment, we have a list of coordinators as subsections, then another subsection with a list of coordinators - at the same header level. 

To me this doesn't seem right and it might be better to put the examples of each coordinator directly into the coordinator section. Another alternative could be to put the list at the top of the coordinator section (ie under the higher level "Coordinators" subsection. I would suggest possibly a table format as well, with the dongle name (ie the manufacturer, and the coordinator type, and then possibly a link for more information). Your current list is just a list of coordinators, but it doesn't tell people how to configure them (ie when they add the coordinator, they still don't know that a Qivicon stick uses the Telegesis driver).

Either way is fine with me, but somehow I think we need to link the dongle types, to the coordinator sections...

Cheers
Chris

Reply:
I also thought about adding them directly for each section but thought that would make it harder to compare sticks. Also since I currently have not used the binding before, I did not know much about configuration. Then I will probably go with a table.

Reply:
Regarding the position: It does not really make much sense to have the examples before introducing the supported coordinator types. If they are listed in a table they have to come afterwards, so a headline is needed to separate them. The level seems to be correct, since it is a subsection of coordinators. One could only create another subsection the containing the three different coordinator type headlines.